docs/help/installation.rst

```diff
@@ -0,0 +1,41 @@
+============
+Installation
+============
+
+Use pip to install any pyfar package. For example the base package *pyfar* is installed by
+
+.. code-block:: console
+
+    $ pip install pyfar
+
+(Requires Python 3.8 or higher)
+
+Reading and writing audio files is supported through `SoundFile`_, which is based on `libsndfile`_. On Windows and OS X, it will be installed automatically. On Linux, you need to install libsndfile using your distribution’s package manager, for example
+
+.. code-block:: console
+
+    $ sudo apt-get install libsndfile1
+
+
+Best practice
+=============
+
+We recommend to install pyfar into a python environment to avoid conflicts with already installed packages. The example below uses the `Anaconda`_ package manager. To create a new environment run
```

Comment:
```suggestion
We recommend to install pyfar into a virtual python environment to avoid conflicts with already installed packages. The example below uses the `Anaconda`_ package manager. To create a new environment run
```